xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -885,6 +920,42 @@ void CXBMCApp::UpdateSessionState()
 
 void CXBMCApp::OnPlayBackStarted()
 {
+    CLog::Log(LOGDEBUG, "%s", __PRETTY_FUNCTION__);
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)